src/main/java/com/powsybl/openloadflow/network/impl/LfBranchImpl.java

```diff
@@ -77,6 +77,9 @@ private static LfBranchImpl createTransformer(TwoWindingsTransformer twt, LfBus
             }
 
             if (controlledSide != null) {
+
+                int currentPosition = ptc.getTapPosition();
```

Comment:
It should not be done like this, network has not to be modified at this stage, even temporarily.